plugins/server-stats/frontend/public/templates/data-points.html

```diff
@@ -1,4 +1,4 @@
-<div class="widget">
+<div id="tod" class="widget">
```

Comment:
Would this indicate it requires styles for times-of-day?
if yes, then plugin should have its own styles defined in it's own css file
if no, let's rename it, not to clash with times of day :)